enhancements/profile-versioning.md

```diff
@@ -0,0 +1,413 @@
+1.3.0 to 1.4.0).
+
+### Version Skew Strategy
+
+Not applicable to this particular enhancement given the Compliance Operator is
+a single installation within a single cluster.
+
+### Operational Aspects of API Extensions
+
+Operational impact on this change should be minimal considering the profiles
+need to be parsed regardless, and this change is limited to setting one
+additional property on a CRD.
+
+The compliance content shipped with the Compliance Operator by default may
+change, which will still be true with this enhancement. Users should expect to
+use `Profile` CRDs in the same way they have in the past.
+
+#### Failure Modes
+
+The `Profile` version attribute should be optional. This will ensure that
+profiles without a version are not required to have one when the content is
+parsed by the operator. If and when the profile is versioned, it will
+automatically advertise the version upon upgrade of the Compliance Operator.
+
+#### Support Procedures
+
+In the case above, users supplying their own content must ensure they're
+setting the profile version in the data stream. Otherwise, they will not be
+able to use custom content with the compliance operator.
+
+Verbose logging will be added to the profile parser to detail the `Profile`
+CRDs being created, and the versions parsed from the data stream so these
+failure scenarios are traceable, and easy to detect.
+
+## Implementation History
+
+Major milestones in the life cycle of a proposal should be tracked in `Implementation
+History`.
+
+1. Update each profile version for OpenShift profiles in [ComplianceAsCode/content](https://github.com/ComplianceAsCode/content)
+2. Implement a new property called `version` for the `Profile` CRD
+3. Add a CI end-to-end test that fails if a profile does not contain a `version`
+4. Document profile life cycles across content and the operator, including versioning
+
+## Drawbacks
+
+A potential drawback is how to best maintain versions across profiles, or if
+single profiles should have multiple versions. These questions will come with
+additional maintenance load for existing compliance content.
+
+To counter this, the `version` property should at the very least match the
+benchmark, as noted in the goals section above. In more complicated scenarios,
+the version could be expanded to handle changes between content, making it
+easier for users to see changes between Compliance Operator updates. In this
+case, version evaluation should be backwards compatible (e.g., `1.4.0` is older
+than `1.4.0-1`), which can be addressed in a subsequent enhancement.
+
+## Alternatives
+
+An alternative to this versioning proposal is to have the `Profile` CRD support
+multiple versions of the same profile, with a list of versions. This would require more complex changes to the `Profile` CRD to support merging to profiles from the same datastream into a single `Profile` object, with multiple versions.
```

Comment:
Nit: this line is very long.

```suggestion
multiple versions of the same profile, with a list of versions. This would require more complex changes to the `Profile` CRD to support merging of profiles from the same data stream into a single `Profile` object, with multiple versions.
```